Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY --link . .
 FROM build-go AS build-enduro
 RUN --mount=type=cache,target=/go/pkg/mod \
 	--mount=type=cache,target=/root/.cache/go-build \
-	go build -o /out/enduro .
+	go build -o /out/enduro ./cmd/enduro-ctl
```

Comment:
I think ideally we want to build the binaries with the version flags, e.g. 
```
    go build -trimpath -o /out/enduro \
    -ldflags="-X '${VERSION_PATH}.Long=${VERSION_LONG}' -X '${VERSION_PATH}.Short=${VERSION_SHORT}' -X '${VERSION_PATH}.GitCommit=${VERSION_GIT_HASH}'" \
    ./cmd/enduro-ctl
```

This would require some more work to get the version info from `build_dist.sh shellvars`.

Reply:
That is a good point I will work on that